UPGRADING.md

```diff
@@ -1,6 +1,70 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 1.3.4
+
+#### Reworking stream and file and un-deprecating stream like-objects
+
+Previously in 0.16 stream-like objects were deprecated. This release restores their functionality for use-cases other than file streaming.
+
+This release also renamed `file` to `sendfile` to better document it's purpose.
+
+To deliver a file via `sendfile` if available do this.
```

Comment:
"if available" doesn't make sense to me, I would just remove

to deliver "in chunks" or something similar?

Reply:
@dblock look at how I re-worded it.. as it is not really "in chunks" as if the web server you are running through supports sendfile then it hands it off to that functionality.